airbyte-integrations/connectors/source-chargebee/source_chargebee/schemas/customer.json

```diff
@@ -431,6 +440,7 @@
       "type": ["null", "array"],
       "items": {
         "type": ["null", "object"],
+        "additionalProperties": true,
```

Comment:
No need to add additionalProperties: true for each object. this is a default behaviour

Reply:
Removed in latest commit. Thanks for the insight.